common/api-review/remote-config-types-exp.api.md

```diff
@@ -0,0 +1,48 @@
+## API Report File for "@firebase/remote-config-types-exp"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export type FetchStatus = 'no-fetch-yet' | 'success' | 'failure' | 'throttle';
+
+// @public
+export type LogLevel = 'debug' | 'error' | 'silent';
+
+// @public
+export interface RemoteConfig {

```

Comment:
Thinking: all the modular methods have been extracted from this interface and are now documented in common/api-review/remote-config-exp.api.md